tests/fixtures.py

```diff
@@ -9,16 +9,16 @@
 
 class ImgeeTestCase(unittest.TestCase):
     def get_test_user(self, name='testuser', uid=1):
-        u = User.query.filter_by(username=unicode(name)).first()
+        u = User.query.filter_by(username=str(name)).first()
         if not u:
             userid = ''.join(random.sample(string.letters, 22))
             u = User(
-                username=unicode(name),
-                userid=unicode(userid),
-                lastuser_token_scope=u'id email organizations',
-                lastuser_token_type=u'bearer',
-                lastuser_token=u'last-user-token',
-                fullname=unicode(name.capitalize()),
+                username=str(name),
+                userid=str(userid),
```

Comment:
These `str` calls are unnecessary. They were used in the old code to avoid SQLAlchemy warnings about passing in strings to unicode columns. Now they are all strings.